pkg/networkmanager/network_neighbors.go

```diff
@@ -54,7 +54,7 @@ func generateNetworkNeighborsNameFromWlid(parentWlid string) string {
 
 func generateNetworkNeighborsLabels(workload k8sinterface.IWorkload) map[string]string {
 	return map[string]string{
-		instanceidhandlerV1.ApiVersionMetadataKey:      workload.GetApiVersion(),
+		//instanceidhandlerV1.ApiVersionMetadataKey:      workload.GetApiVersion(), FIXME invalid label, move to annotations
```

Comment:
is this a fix for all our crds?

Reply:
This should be a discussion between David and Eran